### PR TITLE
[sdb] Add support for function pointers.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by vargaz** [comment]
@monojenkins build failed

File: mcs/class/Mono.Debugger.Soft/Test/dtest.cs [hunk 1]
```diff
@@ -71,8 +71,8 @@ void StackTraceDump (Event e)
 #if !MONODROID_TOOLS
 	Diag.ProcessStartInfo CreateStartInfo (string app, string method = null, string runtimeParameters = null) {
 		var pi = new Diag.ProcessStartInfo ();
-		pi.RedirectStandardOutput = true;
-		pi.RedirectStandardError = true;
+		//pi.RedirectStandardOutput = true;
+		//pi.RedirectStandardError = true;
```
Comment (thaystg) [inline mcs/class/Mono.Debugger.Soft/Test/dtest.cs:75]: This is unrelated?

Comment (vargaz) [reply mcs/class/Mono.Debugger.Soft/Test/dtest.cs]: It was. Although those lines makes it really hard to debug problems since they had output from the runtime.

**Comment by vargaz** [comment]
@monojenkins build failed

**Review by lambdageek — APPROVED** [review]
